Src/FluentAssertions/Collections/GenericCollectionAssertions.cs

```diff
@@ -3308,4 +3468,6 @@ private static int IndexOf(IList<T> items, T item, int index, Func<T, T, bool> c
 
         return -1;
     }
+
+    private static bool SubjectIndexIsConsecutive(int subjectIndex, int previousSubjectIndex) => subjectIndex != previousSubjectIndex + 1;
```

Comment:
♻️ Maybe by changing this into an extension method would make it more readable, e.g. `int1.IsConsecutiveWith(int2)`
🤔 Isn't this implementation doing the opposite? It is consecutive if the indexes only differ by 1, right?

Reply:
Yes, that was me doing things after travelling all day 😅 Will be fixed in the next commit

Reply:
For changing the function to an extension. How do you envision this? Looking at the codebase in the extension folder I see public extensions, but I'm assuming that this isn't something that you want to change the public interface?

Personally, I am not sure that I agree with making it an extension (for something that is used in two places in 1 class) - but will change if you let me know where you want it.

Reply:
> For changing the function to an extension. How do you envision this? Looking at the codebase in the extension folder I see public extensions, but I'm assuming that this isn't something that you want to change the public interface?

As an `internal` extension under `Common`

> Personally, I am not sure that I agree with making it an extension (for something that is used in two places in 1 class) - but will change if you let me know where you want it.

I'm always looking for ways to make code more self-explanetory

Reply:
Thanks for the pointer of where to put it. I'm going to try and see how it looks.

Reply:
Changed